### PR TITLE
Update for promotions

### PR DESCRIPTION
**Pull request by IKarbowiak**
- Split discounts page into separate pages for promotions, vouchers and legacy sales.
   👉 Check `developer/discounts` directory

Resolves saleor/saleor#13438

### CODE REVIEW

**Review by fowczarek — APPROVED** [review]

**Comment by aniav** [comment]
@IKarbowiak this seems to have too many changes, I think. Needs rebasing?

**Review by jakubkuc — APPROVED** [review]

**Review by michal-macioszczyk — APPROVED** [review]

File: docs/upgrade-guides/3-16-to-3-17.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Upgrading from 3.16 to 3.17
+sidebar_position: 8
+---
+
+:::info
+To follow the zero-downtime strategy when upgrading to 3.17, **It is recommended to first migrate to at least 3.16.2** and turn on the Celery worker to process all data migrations asynchronously.
```
Comment (2can) [inline docs/upgrade-guides/3-16-to-3-17.mdx:7]: @IKarbowiak I think you should link to the [Zero downtime migrations](https://docs.saleor.io/docs/3.x/developer/community/zero-downtime-migrations)

**Review by 2can — APPROVED** [review]

File: docs/upgrade-guides/3-16-to-3-17.mdx (hunk 1 above)
Comment (IKarbowiak) [reply docs/upgrade-guides/3-16-to-3-17.mdx:7]: IMO we shouldn't as this page is for developers about writing zero downtime migrations, it seems useless for someone that want to do the upgrade.

Comment (IKarbowiak) [reply docs/upgrade-guides/3-16-to-3-17.mdx:7]: Maybe we are missing page describing what zero-downtime strategy is

**Review by 8r2y5 — APPROVED** [review]

File: docs/upgrade-guides/3-16-to-3-17.mdx (hunk 1 above)
Comment (2can) [reply docs/upgrade-guides/3-16-to-3-17.mdx:7]: https://github.com/saleor/saleor-docs/issues/965

**Pull request merged**